<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_date">
        <property name="text">
         <string>year: 21   mounth: 1   day: 11   week day: 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_time">
        <property name="text">
         <string>hours: 22   minutes: 30   seconds: 30</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_probe_time">
        <property name="text">
         <string>Probe time: 3 sec</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_average">
        <property name="text">
         <string>Average: 10 units</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_humidity">
        <property name="text">
         <string>Humidity: 10.99999 %</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_temperature">
        <property name="text">
         <string>Temperature: 20.99999 C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_adc">
        <property name="text">
         <string>ADC Voltage: 1.99999 V</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_spacer">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_change_time_date">
        <property name="text">
         <string>ok</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_change_time_average">
        <property name="text">
         <string>T and A</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_change_time_average">
        <property name="text">
         <string>ok</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_change_time_average"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_change_time_date">
        <property name="text">
         <string>T and D</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_change_time_date"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_enter_ip">
        <property name="text">
         <string>192.168.1.12:12345</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_enter_ip">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_enter_ip">
        <property name="text">
         <string>IP and P</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
